src/main/java/org/spongepowered/common/mixin/core/tileentity/TileEntityDispenserMixin.java

```diff
@@ -50,4 +56,16 @@
         return new SingleGridLens(0, 3, 3, (Class) TileEntityDispenser.class, slots);
     }
 
+    @Override
+    public void bridge$setDispensedItem(ItemStack dispensed) {
+        this.originalStack = dispensed;
```

Comment:
You do know this is 100% mutable, or are you intending on copying it outside of this?